javascript/node/events/listening-to-events-just-once.md

```diff
@@ -34,32 +28,39 @@ aspects:
 ---
 ## Content
 
-If more than one uncaught exception was to be caught, the applications equivalent to a `closedown` method, where the application is shutdown, would be triggered more than once, consequently causing issues with the applications shutdown procedure.
+Some applications can include a special behavior that happens whenever it is closed. This behavior can be used to free up resources, close a connection to a databases (or anything else), and for the remainder of this insight we will call it `closedown`.
 
-To only call the `closedown`  method once, only the first instance of an uncaught exception being caught should be used. This is done using `emitter.once` , for example, if an application's emitter was to be called *test*:
-```javascript
-test.once(‘uncaughtException’,
-                  function(err) {
+Suppose the application throws an unexpected exception, which is then caught followed by the `closedown` method being called before terminating the application (which is the expected behavior).
+
+Now, let's say the the application throws 5 unexpected exceptions at the same time. This will lead to the `closedown` method being triggered 5 times concomitantly which might lead to issues in the shutdown procedure.
+
+The trigger of this procedure is done through an `uncaughtException` event, and you want a way to listen to this just once.
 
+To achieve this, only the first instance of an uncaught exception being caught should be used. By calling the `.once` method to our emitter (in our case the emitter is called `test`, which would lead to us calling `test.once`):
+
+```javascript
+test.once("uncaughtException", function(
+  err
+) {
   closeApp(function(err) {
     // log error
   });
 
-
   // exit
   test.exit(1);
-
 });
 ```
 
-This doesn’t solve the issue entirely as if two `uncaughtException`s do occur the second will trigger an instant process shutdown, interrupting and overriding the `closedown` method current in progress. To avoid this, log each uncaught exception:
+This doesn’t solve the issue entirely. If two `uncaughtException`s do occur, the second will trigger an instant process shutdown, interrupting and overriding the `closedown` method currently in progress. To avoid this, log each uncaught exception:
```

Comment:
I don't this this information was correct. Logging doesn't affect the number of invocations to this callback, I think we can remove all the text from this paragraph and the code snippet bellow.